group-project/src/main/webapp/style.css

```diff
@@ -111,7 +111,7 @@ h2 {
 @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@900&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap");
 
-#heading {
+#heading, #commentHeading {
   display: flex;
   font-family: Roboto;
```

Comment:
suggestion: font-family: 'Playfair Display', sans-serif;
(so that it will go together with your h2 fonts :D)

Reply:
Ah I see haha I thought this was your css, so I just added #commentHeading to the same style as the prototype. I can work on the style of the page if you want me to👍👍